game-core/src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -265,10 +264,7 @@ boolean createServerMessenger(
       remoteMessenger = new RemoteMessenger(unifiedMessenger);
       remoteMessenger.registerRemote(serverStartupRemote, SERVER_REMOTE_NAME);
       channelMessenger = new ChannelMessenger(unifiedMessenger);
-      final NullModeratorController moderatorController = new NullModeratorController(serverMessenger, null);
-      moderatorController.register(remoteMessenger);
```

Comment:
While this line set up a communication channel for the moderator controller in a game server, it apparently was never used.  The game client (as opposed to the lobby client) never attempts to open this channel.